src/main/java/pro/sky/course2/store/component/Cart.java

```diff
@@ -0,0 +1,26 @@
+package pro.sky.course2.store.component;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.annotation.SessionScope;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@SessionScope
+public class Cart {
+    private final List<Integer> items;
+
+    public Cart() {
+        this.items = new ArrayList<>();
+    }
+
+    public List<Integer> addItems(List<Integer> IDs) {
```

Comment:
переменные лучше именовать маленькими буквами
```suggestion
    public List<Integer> addItems(List<Integer> ids) {
```